roles/marathon/tasks/main.yml

```diff
@@ -9,7 +8,7 @@
     - marathon
 
 - name: create marathon artifact store directory
-  when: marathon_artifact_store_dir is defined and marathon_enabled | bool
+  when: marathon_artifact_store_dir is defined
```

Comment:
Yeah there's a number of other places we have this though - do we want to create a separate issue for that? e.g. - 

```

/workspace/apollo/roles/consul/tasks/main.yml:
   57:   when: consul_join is defined
   71:   when: consul_join is defined

/workspace/apollo/roles/docker/tasks/main.yml:
   37:   when: http_proxy is defined and http_proxy != ''

/workspace/apollo/roles/marathon/tasks/main.yml:
   11:   when: marathon_artifact_store_dir is defined

/workspace/apollo/roles/mesos/defaults/main.yml:
   35: {% if mesos_master_quorum is defined %}

/workspace/apollo/roles/serverspec/tasks/main.yml:
   48:   when: test_role is defined and serverspec_run_tests

/workspace/apollo/roles/zookeeper/defaults/main.yml:
   10: 	{%- if zookeeper_host_list is defined -%}

/workspace/apollo/roles/zookeeper/templates/zoo.cfg.j2:
    6: {% if zookeeper_conf is defined %}
```